content/en/internal/configuration-database/_index.md

```diff
@@ -9,9 +9,36 @@ type: "docs"
 in progress, more documentation is currently available at
 [EGI's wiki](https://wiki.egi.eu/wiki/GOCDB). {{% /pageinfo %}}
 
-The Grid Configuration Database (also know as the GOCDB) contains general
+The EGI Configuration Database (previously known as the GOCDB) contains general
 information about the sites participating to the production infrastructure.
 
+It's a central registry to record information about the topology of the EGI
+e-Infrastructure. This includes entities such as Operations Centres, Resource
+Centres, service endpoints and their downtimes, contact information and roles of
+users responsible for operations at different levels.
+
 It's used by all the actors (end-users, site managers, NGI mangers, support
 teams, VO managers), by other tools and by third party middleware to discover
 information about the infrastructure topology.
+
+## Identity card
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                     | Value                                                                 |
+| ---------------------------- | --------------------------------------------------------------------- |
+| Name                         | Configuration Database                                                |
+| Description                  | Central registry of the infrastructure topology                       |
+| URL                          | https://goc.egi.eu                                                    |
+| Support Email                | gocdb-admin@mailman.egi.eu                                            |
+| Helpdesk Support Unit        | EGI Services and Service components -> Configuration Database (GOCDB) |
```

Comment:
Do we have a link for this?

Reply:
If we had some basic stuff about using GGUS under https://docs.egi.eu/support/ we could point to that.
I don't think (may be wrong, never saw it but never tried) we can easily make an URL allowing to create a ticket for a specific SU.

Reply:
I've added a link to the Helpdesk page (that is a bit rough)